src/edu/washington/escience/myria/operator/Operator.java

```diff
@@ -107,6 +108,30 @@ public ProfilingLogger getProfilingLogger() {
   }
 
   /**
+   *
+   * @return PythonFunctionRegistrar for operator.
+   * @throws DbException  in case of error.
+   */
+  public PythonFunctionRegistrar getPythonFunctionRegistrar() throws DbException {
+    PythonFunctionRegistrar pyFuncRegistrar = null;
+    try {
+      if (execEnvVars.get(MyriaConstants.EXEC_ENV_VAR_TEST_MODE) == null) {
+
+        if (getLocalSubQuery() instanceof WorkerSubQuery) {
+          pyFuncRegistrar =
+              ((WorkerSubQuery) getLocalSubQuery()).getWorker().getPythonFunctionRegistrar();
+        }
+      }
+
+    } catch (Exception e) {
+      LOGGER.debug("exception getting pyFuncRegistrar");
+      pyFuncRegistrar = null;
```

Comment:
Here, `pyFuncRegistrar` will already be null anyway. Also, wouldn't you want to throw an exception at this point? I noticed that at several points in your code, you call this function to get the registrar, but you wouldn't ever want to return a null value right?